cfme/intelligence/reports/menus.py

```diff
@@ -171,6 +172,14 @@ def manage_folder(self, group, folder=None):
             view.save_button.click()
 
 
+@attr.s
+class ReportMenusCollection(BaseCollection):
+    """Collection object for the
+    :py:class:'cfme.intelligence.reports.ReportMenu'."""
```

Comment:
Please checkout napoleon/sphinx guidelines for docblocks, this should be on one line, with the second line empty. This fits in 100 chars, and should just be on a single line.